src/services/milestones/milestones.hooks.js

```diff
@@ -330,6 +331,7 @@ module.exports = {
     create: [
       checkConversionRates(),
       checkMilestoneDates(),
+      checkMilestoneName(),
```

Comment:
We should consider patch and update too, use may edit milestone after he has created.